src/models/user.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+
+const UserSchema = new mongoose.Schema({
+  netId: { type: String, required: true, unique: true },
+  name: { type: String, required: true },
+  created: {
+    type: Date,
+    default: Date.now,
+  },
+  isOfficer: { type: Boolean, default: false },
```

Comment:
do you think we can change this field into an enum for membership type (so that the possible values are officer, committee member, or member)? that way in the future we can add features like checking in for open office

Reply:
checking in for open office sounds great 😮